pkg/cache/clusterqueue.go

```diff
@@ -684,40 +683,53 @@ func (c *ClusterQueue) UsedCohortQuota(fName kueue.ResourceFlavorReference, rNam
 	return cohortUsage
 }
 
-// DominantResourceShare returns a value from 0 to 100 representing the maximum of the ratios
+// DominantResourceShare returns a value from 0 to 1000 representing the maximum of the ratios
```

Comment:
The new computation made one of the existing tests fail because of not enough resolution. I think this should be enough. We could also consider floating numbers in the future, but I thought an integer value could be good enough and easier to debug in most cases.